Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm

```diff
@@ -260,7 +260,7 @@ static void disableDowngradeToLayoutManager()
         // these methods need to be called directly.
         if (CFEqual(key.createCFString().get(), kAXSReduceMotionPreference) && [value isKindOfClass:[NSNumber class]])
             _AXSSetReduceMotionEnabled([(NSNumber *)value boolValue]);
-        else if (CFEqual(key.createCFString().get(), increaseContrastPreferenceKey()) && [value isKindOfClass:[NSNumber class]])
+        else if (CFEqual(key.createCFString().get(), increaseContrastPreferenceKey().createNSString().get()) && [value isKindOfClass:[NSNumber class]])
```

Comment:
Seems a little strange to compare these two by converting to `CFString`; can’t we compare without creating new objects? Also, seems inconsistent to use `createNSString` instead of `createCFString` for the new code.

Reply:
You're right, we can just compare WTF:Strings here. I wasn't thinking, I was in add `createNSString()` to fix the build mode :)